test/gc/Dialect/OneDNNGraph/onednn-graph-ops.mlir

```diff
@@ -11,6 +11,15 @@ func.func @matmul(%arg0: tensor<128x512xbf16>, %arg1: tensor<512x64xbf16>,
   return %0 : tensor<128x64xbf16>
 }
 
+// CHECK-LABEL: @matmul_broadcast_batch
+func.func @matmul_broadcast_batch(%arg0: tensor<6x128x512xbf16>, %arg1: tensor<4x1x512x64xbf16>, 
+                  %arg2: tensor<64xbf16>) -> tensor<4x6x128x64xbf16> {
+  // CHECK: onednn_graph.matmul 
+  %0 = onednn_graph.matmul %arg0, %arg1, %arg2 
```

Comment:
Add concrete shape check for such special case?

Reply:
This will check for shape match error, if the tensor shape does not match the op, error will be produced and test will fail. onednn_graph.matmul will only check for shape compatibility, and insert broadcast when lowering.